src/overlays/actors/ovl_En_Hgo/z_en_hgo.c

```diff
@@ -107,59 +118,59 @@ void EnHgo_Destroy(Actor* thisx, PlayState* play) {
     Collider_DestroyCylinder(play, &this->collider);
 }
 
-void func_80BD03EC(EnHgo* this) {
+void EnHgo_SetupIdle(EnHgo* this) {
     this->actor.flags &= ~ACTOR_FLAG_1;
-    this->actionFunc = func_80BD0410;
+    this->actionFunc = EnHgo_Idle;
 }
 
-void func_80BD0410(EnHgo* this, PlayState* play) {
+void EnHgo_Idle(EnHgo* this, PlayState* play) {
```

Comment:
Probably fine as is, but perhaps this should be `DoNothing`. Usually when I see `Idle` I kind of expect some logic for what/how it is waiting for something, but here there is nothing.

Reply:
The actor doesn't have much logic for how to wait since he is just standing there waiting to interact, so I went with Idle, but I do also see what you mean about `DoNothing` since that's what's happening until you talk with him...

Reply:
I think for me mostly that waiting logic is usually in idle itself (I don't remember seeing a completly empty Idle function), even if the actor is physically doing nothing. For example `BgCraceMovebg_OpeningDoor_Idle` is simple, just idle logic for a switchflag, but is physically doing nothing. This actor is different in that his idle logic is all just in Update, which all action functions would use. 
Again, this fine as it is if you want.